modules/dnn/test/test_backends.cpp

```diff
@@ -161,9 +163,11 @@ TEST_P(DNNTestNetwork, MobileNet_SSD_Caffe)
         throw SkipTestException("");
     Mat sample = imread(findDataFile("dnn/street.png", false));
     Mat inp = blobFromImage(sample, 1.0f / 127.5, Size(300, 300), Scalar(127.5, 127.5, 127.5), false);
+    float l1 = (target == DNN_TARGET_OPENCL_FP16) ? 0.0007 : 0.0;
```

Comment:
Please replace to `backend == DNN_BACKEND_DEFAULT && target == DNN_TARGET_OPENCL_FP16` .

Reply:
fixed, thank you